### PR TITLE
[DRAFT] Markdown Linting

### PR DESCRIPTION
**Pull request by SharonHart**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -31,7 +31,18 @@ jobs:
...
+      displayName: 'Markdown linting validation'
+      steps:
+        - task: Bash@3
+          displayName: 'markdown lint'
+          inputs:
+            targetType: 'inline'
+            script: |
+              npm install -g markdownlint-cli
+              markdownlint .
+
```
Comment (SharonHart) [inline azure-pipelines.yml:45]: Will be extracted to a template

File: docs/deployment-samples/deploy-to-app-service.md [hunk 2]
```diff
@@ -55,26 +64,40 @@ LOG_ANALYTICS_WORKSPACE_RESROUCE_GROUP=<resource group of log analytics>
 LOG_ANALYTICS_WORKSPACE_NAME=<log analytics name>
 
 # create a log analytics workspace
-az monitor log-analytics workspace create --resource-group $LOG_ANALYTICS_WORKSPACE_RESROUCE_GROUP --workspace-name $LOG_ANALYTICS_WORKSPACE_NAME
+az monitor log-analytics workspace create \
+--resource-group $LOG_ANALYTICS_WORKSPACE_RESROUCE_GROUP \
+--workspace-name $LOG_ANALYTICS_WORKSPACE_NAME
```
Comment (SharonHart) [inline docs/deployment-samples/deploy-to-app-service.md:69]: All of the bash scripts needs to be verified to avoid regressions

**Comment by omri374** [comment]
Looks good! Consider adding a section on linting to CONTRIBUTING.md + links for setting up auto-linting on IDEs so that it would be easier to be aligned with the linter.

**Review by TessFerrandez — APPROVED** [review]
Looks good

**Comment by SharonHart** [comment]
Closing since REDAMEs changes fro  MKdocs won't lint.

**Pull request closed without merging**